changelog/unreleased/enhancement-add-sse-event-for-moving

```diff
@@ -0,0 +1,7 @@
+Enhancement: Add SSE event for moving
+
+We've added Server-Sent Events (SSE) for moving resources.
+This enhancement ensures that the moving action instantly propagated to other browsers and devices where the logged in user has access the relevant files and folders.
+
```

Comment:
I think this is pretty technical and not something a regular user reading the changelog would understand...